recipes-bsp/u-boot/u-boot-qoriq_2014.07.bb

```diff
@@ -19,6 +19,20 @@ inherit deploy
 
 SRC_URI = "git://git.freescale.com/ppc/sdk/u-boot.git;nobranch=1"
 SRCREV = "659b6a23a8b1f3026200bc6352dbacef53f4dcb1"
+SRC_URI += "file://0001-Chage-NOR-flash-szie-to-16M.patch \
```

Comment:
Its probably safer to change this to SRC_URI_append_t600 in case this version is every used with other machine types (I believe the patches are t600 specific and some are applying to parts of u-boot that may be of value to other machine types anyways).